languages/clojure/exercises/concept/floating-point-numbers/.docs/introduction.md

```diff
@@ -0,0 +1,8 @@
+A floating-point number is a number with zero or more digits behind the decimal separator. Examples are `-2.4`, `0.1`, `3.14`, `16.984025` and `1024.0`.
+
+Different floating-point types can store different numbers of digits after the digit separator - this is referred to as its precision. This means that trying to store PI in a `single` will only store the first 6 to 9 digits (with the last digit being rounded).
```

Comment:
@porkostomus `store PI in a `single` `, sorry what does this 'single' mean, is there a data type called single? (I'm quite new to clojure...)

Reply:
I had the same question. The list of types below only mention `Double` and `BigDecimal`.